beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/singlepeer/SyncManager.java

```diff
@@ -185,11 +187,16 @@ public void unsubscribeFromSyncChanges(final long subscriberId) {
   public SyncingStatus getSyncStatus() {
     final boolean isSyncActive = isSyncActive();
     if (isSyncActive) {
-      Optional<Eth2Peer> bestPeer = findBestSyncPeer();
+      final Optional<Eth2Peer> bestPeer = findBestSyncPeer();
       if (bestPeer.isPresent()) {
-        UInt64 highestSlot = bestPeer.get().getStatus().getHeadSlot();
+        final PeerStatus bestPeerStatus = bestPeer.get().getStatus();
         return new SyncingStatus(
-            true, recentChainData.getHeadSlot(), peerSync.getStartingSlot(), highestSlot);
+            true,
+            recentChainData.getHeadSlot(),
+            peerSync.getStartingSlot(),
+            new SyncingTarget(
+                new SlotAndBlockRoot(bestPeerStatus.getHeadSlot(), bestPeerStatus.getHeadRoot()),
+                1));
```

Comment:
the const 1 here might be a bit misleading if it comes out in one of the logs, should we print the total amount of peers instead even though they might not all have the same target?

Reply:
This has been removed